src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,78 @@
-import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[];
+  selectedTodo: Todo | null;
+  setSelectedTodo: (todo: Todo) => void;
```

Comment:
The `setSelectedTodo` function should allow setting the state to `null` to close the modal, as required by the task: 'x button should close the modal'. Update the type definition to allow `null` values.